pages/categories/index.js

```diff
@@ -0,0 +1,95 @@
+import CorrectIcon from "@/assets/icons/CorrectIcon";
+import EditIcon from "@/assets/icons/EditIcon";
+import AddCategoryForm from "@/components/AddCategoryForm";
+import Category from "@/components/Category";
+import Heading from "@/components/PageHeading";
+import { MainContent } from "@/components/StyledComponents/MainContent";
+import { StyledSecondaryButton } from "@/components/StyledComponents/StyledButtons";
+import {
+  StyledCenterAlign,
+  StyledSection,
+  StyledSectionRightAlign,
+} from "@/components/StyledComponents/StyledSection";
+import { hasToken } from "@/utils/checkUser";
+import { useRouter } from "next/router";
+import { useState } from "react";
+
+export async function getServerSideProps(context) {
+  const token = await hasToken(context.req);
+
+  if (!token) {
+    return {
+      redirect: {
+        destination: "/login",
+        permanent: false,
+      },
+    };
+  }
+
+  return { props: {} };
+}
+
+export default function Categories({
+  handleAddCategory,
+  categoryData,
+  categoryMutate,
+  wordList,
+  activeUser,
+  databaseMutate,
+}) {
+  const [isEditMode, setIsEditMode] = useState();
+  const router = useRouter();
+  return (
+    <MainContent>
+      <Heading id="add-form-title" PageTitle="Your Categories" />
+      <StyledSectionRightAlign>
+        <StyledSecondaryButton type="button" onClick={() => router.back()}>
+          Back
+        </StyledSecondaryButton>
```

Comment:
this should be a Link component

Reply:
or do you want specifically that the user is sent to the previous page?

Reply:
Yes, I wanted to go back to the previous page, whichever one it was. Because there are different ways to get to this page, and I don't think it needs to be in the nav just yet.

Reply:
... buuuuut **very** strictly speaking a link outside of a navigation is still a link, regardless. That's the a11y way. Here's how to handle it in combination with [useRouter](https://nextjs.org/docs/pages/api-reference/functions/use-router). You'd extract the `href` and define a click handler.

Reply:
Okay. :) I'll convert it into a Link